app/mailers/csv_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class CsvMailer < ApplicationMailer
+  def created_email(csv_export)
+    address = csv_export.email
+    subject = "Samson CSV Export Completed"
+    url = csv_url( id: csv_export.id, format: 'csv')
+    body = "Download your CSV file at #{url}"
+    mail(to: address, subject: subject, body: body) unless (address.nil? or address.empty?)
```

Comment:
handle the `email.blank?` case outside, I think otherwise it creates a strange empty email / no need to create a job